mmeds/CSS/stylesheet.css

```diff
@@ -3198,3 +3198,6 @@ button:disabled,
 
 /* Change the background color of the dropdown button when the dropdown content is shown */
 .dropdown:hover .dropbtn {background-color: #3e8e41;} 
+
+/*Hide the dual barcodes dropdown if metadata_type is Subject */
+div.Subject {display: none;}
```

Comment:
Move to the top of the stylesheet (that's where the other MMEDS specific edits are)